lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -1746,8 +1727,16 @@ class TopologicalSort {
   TopologicalSort(const SPIRVTypeVec &TypeVec,
                   const SPIRVConstantVector &ConstVec,
                   const SPIRVVariableVec &VariableVec,
-                  const SPIRVForwardPointerVec &ForwardPointerVec)
-      : ForwardPointerVec(ForwardPointerVec),
+                  SPIRVForwardPointerVec &ForwardPointerVec)
```

Comment:
Why it's no longer a const?

Reply:
Because it is now modified in this function, see line 1754

Reply:
Indeed